Telerik.JustMock/Expectations/CommonExpectation.cs

```diff
@@ -405,6 +405,29 @@ public IOccurrence InOrder(string message = null)
 				});
 		}
 
+		/// <summary>
+		/// Determines wheter prerequisite is met
+		/// </summary>
+		public bool IsMet
+		{
+			get
+			{
+				return ProfilerInterceptor.GuardInternal(() => ((IMethodMock)this).IsUsed);
+			}
+		}
+
+		/// <summary>
+		/// Specifies a call should occur only after all of the given prerequisites have been called.
```

Comment:
This summary should be changed to:
Specifies that a call should occur only after all of the given prerequisites have been met.